openxla_patches/grpc_version.diff

```diff
@@ -1,64 +1,8 @@
 # Tensorflow's patch to update the protobuf will regress the XRT runtime.
```

Comment:
this is for XRT only

Reply:
thanks for pointing, test in https://github.com/pytorch/xla/pull/5222

Reply:
https://github.com/pytorch/xla/pull/5222 has passed all tests, will apply this change

Reply:
Thanks, updated